control-plane/grpc/proto/v1/volume/volume.proto

```diff
@@ -57,8 +57,14 @@ message VolumeSpec {
   bool thin = 9;
   // Affinity Group related information.
   optional AffinityGroup affinity_group = 10;
-  // Indicates if this volume was created as a clone.
-  bool is_clone = 12;
+  // Volume Content Source i.e the snapshot and its source volume.
+  optional CloneContentSource content_source  = 11;
+
+  // Volume Content Source i.e the snapshot and its source volume.
+  message CloneContentSource {
+    google.protobuf.StringValue snapshot = 1;
```

Comment:
optional string ?

Reply:
Kept it similar to what we already use for uuid.